src/renderer/components/+network-services/service-port-component.tsx

```diff
@@ -61,14 +105,32 @@ export class ServicePortComponent extends React.Component<Props> {
   render() {
     const { port } = this.props;
 
+    if (this.forwardPort == null) {
+      this.forwardPort = port.port;
+    }
+
+    const portForwardAction = async () => {
+      if (this.isPortForwarded) {
+        await this.stopPortForward();
+      }else {
+        await this.portForward();
+      }
+    };
+
     return (
       <div className={cssNames("ServicePortComponent", { waiting: this.waiting })}>
-        <span title="Open in a browser" onClick={() => this.portForward() }>
-          {port.toString()}
-          {this.waiting && (
-            <Spinner />
-          )}
-        </span>
+        {port.toString()}
+        {" "}
+        <text>to</text>
+        <Input className={"portInput"}
+          type="number"
+          min="0"
```

Comment:
```suggestion
          min="1"
```
I think for TCP, port number 0 is reserved and cannot be used. I guess it's debatable if any of the low numbers should be used

Reply:
I have used the min port as 0, as kubectl allocates random port for input 0, Thanks for your input